### ISSUE TITLE
animate() for tuples not respecting tween

### ISSUE THREAD

**Issue by lordmauve** [lordmauve/wasabi2d#36]
I tried to write an animation on pos with tween=`bounce_end`.

However I see no bounces; I saw a very quick (linear?) tween to the target position.

```
import wasabi2d as w2d                                                          
                                                                                  
  scene = w2d.Scene(background="#223366")                                         
                                                                                  
  cx, cy = scene.width // 2, scene.height // 2                                    
                                                                                  
  title = scene.layers[0].add_label(                                              
      "Example",                                                 
      pos=(cx, -10),                                                               
      fontsize=48,                                                                
      align='center',                                                             
  )                       

w2d.animate(title, tween='bounce_end', pos=(cx, cy))
w2d.run()
```

**Comment by lordmauve** [comment]
I looked into `bounce_end` for another thing and ultimately decided it was just happening too quickly - over a 1 second tween it reaches 90% of the transition in maybe 2 frames.

**Comment by lordmauve** [comment]
Oh, maybe that was `out_elastic`.

**Comment by encukou** [comment]
This should be fixed in #62.
When I filed #61, I didn't realize it was a duplicate.
